### PR TITLE
Add Homebrew installation option

### PR DESCRIPTION
**Pull request by chenrui333**
Relates to https://github.com/Homebrew/homebrew-core/pull/40619

Adding an installation option for Homebrew users. :)

### CODE REVIEW

**Comment by chenrui333** [comment]
@rakyll ping :)

**Comment by chenrui333** [comment]
what should I do get this merged?

File: README.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ To preserve the name for its original owner, we renamed this project to hey.
 
     go get -u github.com/rakyll/hey
 
+### Using package manager
+
+If you want to use a package manager:
```
Comment (rakyll) [inline README.md]: Can you instead say something like
```
macOS:
-  [Homebrew](https://brew.sh/) users can use `brew install hey`.
```

Comment (chenrui333) [reply README.md]: Sure, I can update it. Thanks!

**Comment by rakyll** [comment]
Thanks!

**Comment by chenrui333** [comment]
My pleasure. :)

**Pull request merged**